src/Frontend/Core/Header/Header.php

```diff
@@ -126,7 +126,7 @@ public function __construct(KernelInterface $kernel)
         $this->meta = new MetaCollection();
 
         // add some default CSS files
-        $this->addCSS('/src/Frontend/Core/Layout/Css/screen.css');
+        $this->addCSS('/src/Frontend/Themes/' . Theme::getTheme() . '/Core/Layout/Css/screen.css');
```

Comment:
Aah noticed a big difference between the `/src/Frontend/Core/Layout/Css/screen.css` in 4 and 5. Screen.css in 4 does not contain any specific code:
```
/*
    Stylesheet for <>

    Media: screen, projection
    Copyright: Company <http://www.company.com>
    author: Name <name@domain.com>
*/

@media print {
    /*
        Stylesheet for <>

        Media: print
        Copyright: Company <http://www.company.com>
        author: Name <name@domain.com>
    */
}
```

While in version 5 it contains really specific code. Which is not wanted when you are developing your own theme.